src/Modules/Pages/Domain/RevisionBlock/RevisionBlockType.php

```diff
@@ -60,7 +65,9 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'class' => Block::class,
                 'required' => false,
                 'group_by' => static fn (Block $block): TranslationKey => $block->getBlock()->getModule()->asLabel(),
-                'choice_label' => static fn (Block $block): TranslationKey => $block->getLabel(),
+                'choice_label' => function (Block $block): string {
```

Comment:
could be an arrow function that is not static to fit in more with the rest of the code

Reply:
done: https://github.com/forkcms/forkcms/pull/3555/commits/0eee4916220942e1607775ea492552d6a300601a